src/main/src/main/java/org/geoserver/systemstatus/MetricValue.java

```diff
@@ -112,7 +105,11 @@ public String getValueUnit() {
             return BaseSystemInfoCollector.DEFAULT_VALUE;
         }
         if (value instanceof Double || value instanceof Float) {
-            return String.format("%.2f %s", value, unit);
+            final Number numberValue = (Number) value;
+            return String.format(
```

Comment:
Why this change?

Reply:
As far I remember this was due to automatic error prone maven checking warning.